manifest.yml

```diff
@@ -6,7 +6,7 @@ display_information:
 features:
   app_home:
     home_tab_enabled: true
-    messages_tab_enabled: false
+    messages_tab_enabled: true
```

Comment:
Re-enabling this so users can visit the messages tab to see old reminders. Still read-only.